manager/recurringjob.go

```diff
@@ -114,9 +114,64 @@ func (m *VolumeManager) UpdateRecurringJob(spec longhorn.RecurringJobSpec) (*lon
 }
 
 func (m *VolumeManager) DeleteRecurringJob(name string) error {
+	if err := m.CleanVolumeRecurringJob(name); err != nil {
```

Comment:
Should we clean up in the controller instead to react for the recurring job deletion instead of just tackling in the same call?

Reply:
Thanks, that makes more sense. Addressed.